CONTRIBUTING.md

```diff
@@ -224,6 +225,54 @@ GitHub has a [nice introduction](https://help.github.com/articles/github-flow/)
 <img align="center" src="https://i.imgur.com/s8yELfK.png" alt="Example-Contribution" width="800"/>
 
 
+## Updating the schema
+
+Portions of the BIDS specification are defined using YAML files, in order to make the specification machine-readable.
+Currently, the only portion of the specification that relies on this schema is the Entity Table, but any changes to the specification should be mirrored in the schema.
+
+### The format of the schema
+
+The schema reflects the files and objects in the specification, as well as associations between these objects.
+Here is a list of the files and subfolders of the schema, with a description for each:
+
+- `associated_data.yaml`:
+    Folders that contain files falling outside the jurisdiction of the specification, but which are commonly contained within the same folder as a BIDS dataset.
+    The schema specifies which folders are accepted and whether they are required or optional.
+- `top_level_files.yaml`:
+    Modality-agnostic files stored at the top level of a BIDS dataset.
+    The schema specifies whether these files are required or optional, as well as acceptable extensions for each.
+- `modalities.yaml`:
+    Modalities supported by the specification, along with a list of associated datatypes.
```

Comment:
AFAIK, "datatype" is not a thing in the spec. (e.g. I just searched the spec and found no reference to "datatype".  Can you give an explanation here or, at the top of the list, what is meant by "datatype"?

Reply:
Data types fall under the common principles. They correspond to the folders (`anat`, `func`, `meg`, etc.). I can split the word in the text to make it clearer, and maybe link back to the relevant part of common principles?

Reply:
Ah!  "data type" two words!  Yes, of course.